.github/workflows/release-dev.yaml

```diff
@@ -6,26 +6,6 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+-*"
 
 jobs:
-  # release_creation:
```

Comment:
Should we keep commented  code?

Reply:
I guess we don't need it anymore since we already have another flow for creating a release. But if you thin it should be left, then it's fine with me.

Reply:
up to you ;)